tests/pipeline/default/sgdemux.nf.test

```diff
@@ -0,0 +1,25 @@
+nextflow_pipeline {
+
+    name "Test SGDEMUX"
+    script "main.nf"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                input = "$baseDir/assets/inputs/sgdemux-samplesheet.csv"
+                demultiplexer = 'sgdemux'
+                outdir = "$outputDir"
+            }
+        }
+
+        then {
+            assert workflow.success
+            // assert workflow.trace.succeeded().size() == 8
```

Comment:
TODO

Reply:
Updated in commit 5b5e78e63c3717ab228c5dacb73df6bf2960ee5d